### PR TITLE
Implement web.run_app utility function

### PR DESCRIPTION
**Pull request by asvetlov**
Fix for #690

Let's delay `python -m aiohttp.web HOST PORT pkg.module.main [ARGS]` for a while.

I've created #733 for that.

The PR adds `run_app` utility function along with docs and tests.

### CODE REVIEW

File: aiohttp/web.py [hunk 1]
```diff
@@ -287,3 +287,38 @@ def __call__(self):
...
+def run_app(app, *, host='0.0.0.0', port=None, loop=None,
+            shutdown_timeout=60.0, ssl_context=None):
+    """Run an app locally"""
+    if port is None:
+        if not ssl_context:
+            port = 8080
+        else:
+            port = 8443
+
+    if loop is None:
```
Comment (jashandeep-sohi) [inline aiohttp/web.py]: Is an explicit `loop` parameter necessary -- Wouldn't `loop = app.loop` suffice here?
Is there a use case where the App would run on a different loop than the Server?

Comment (asvetlov) [inline aiohttp/web.py]: Good point!

**Comment by rutsky** [comment]
LGTM

File: aiohttp/web.py [hunk 2]
```diff
@@ -287,3 +287,37 @@ def __call__(self):
...
+            port = 8443
+
+    loop = app.loop
+
+    handler = app.make_handler()
+    srv = loop.run_until_complete(loop.create_server(handler, host, port,
+                                                     ssl=ssl_context))
+
+    scheme = 'https' if ssl_context else 'http'
+    prompt = '127.0.0.1' if host == '0.0.0.0' else host
```
Comment (popravich) [inline aiohttp/web.py:308]: How should someone bind to public interface?

Comment (popravich) [inline aiohttp/web.py:308]: sorry, read from end to top))

Comment (kxepal) [inline aiohttp/web.py:308]: Use `::` for IPv6 (:

Comment (asvetlov) [inline aiohttp/web.py:308]: Sorry, I don't follow.
Isn't knowledge of host/port info enough?

Comment (popravich) [inline aiohttp/web.py:308]: As I already said -- I read diff from bottom to top) thought that you replaced '0.0.0.0' with localhost and bind to it)

**Pull request merged**